xesmf/backend.py

```diff
@@ -381,9 +383,22 @@ def esmf_regrid_build(
         If False (default), raise error if grids contain degenerated cells
         (i.e. triangles or lines, instead of quadrilaterals)
 
+    vector_regrid : bool, optional
+        If True, treat a single extra (non-spatial) dimension in the source and
+        destination data fields as the components of a vector. (If True and
+        there is more than one extra dimension in either the source or
+        destination data fields, an error will be raised.) If not specified,
+        defaults to False.
+
+        Only vector dimensions of size 2 are supported. The first entry is
+        interpreted as the east component and the second as the north component.
+        i.e., ``extra_dims`` must be ``[2]``.
+
+        Requires ESMPy 8.9.0 or newer.
+
     Returns
     -------
-    grid : ESMF.Grid object
+    regrid : ESMF.Regrid object
```

Comment:
This documentation change of the return value is unrelated to the other changes in this PR; I just noticed the error when working on these changes.